exercises/practice/square-root/square-root.spec.coffee

```diff
@@ -0,0 +1,40 @@
+
+SquareRoot = require './square-root'
+
+describe 'SquareRoot', ->
+  sqrt = new SquareRoot()
```

Comment:
The initial code:
```
class SquareRoot
  @squareRoot: (radicand) ->

module.exports = SquareRoot
```

produces the following error:
`TypeError: sqrt.squareRoot is not a function`

Removing the @ from the solution solves the error.